drivers/chown.go

```diff
@@ -0,0 +1,171 @@
+package graphdriver
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/containers/storage/pkg/idtools"
+	"github.com/containers/storage/pkg/reexec"
+)
+
+const (
+	chownByMapsCmd = "storage-chown-by-maps"
+)
+
+func init() {
+	reexec.Register(chownByMapsCmd, chownByMapsMain)
+}
+
+func chownByMapsMain() {
+	if len(os.Args) < 1 {
+		fmt.Printf("requires mapping configuration on stdin and directory path")
```

Comment:
Should this be logrus?  Is the library going to exec chown?

Reply:
Yes, this is invoked via `reexec`.  I want to let the caller use the output text to create an error, so it's preferable that it not include logrus timestamp/level information at this point.

Reply:
Should this be stderr then?